app/assets/javascripts/discourse/app/controllers/preferences/second-factor.js

```diff
@@ -130,8 +156,11 @@ export default Controller.extend(CanCheckEmails, {
 
       this.dialog.deleteConfirm({
         title: I18n.t("user.second_factor.disable_confirm"),
+        message: htmlSafe(this.disableAllModalTemplate()),
```

Comment:
I'm a little uncomfortable with the blanket `htmlSafe` usage here. Would prefer moving this to the helper and only using it if/when there is HTML in the message. (And IMO we don't need the bolds in some of the messages, given that these screens have pretty clear headings now.)